#request
int8 data
int8 START_PROFILING =1
int8 STOP_PROFILING  =2
int8 START_SCANNING  =3
int8 STOP_SCANNING   =4
int8 GET_CAMERA_DATA =5
int8 SAVE_MAP        =100
int8 LOAD_MAP        =101
---
#response
int8 data
string error
int8 ACK      =0
int8 ERROR    =1
int8 DONE     =2

